pkg/gds/tokens/tokens.go

```diff
@@ -166,6 +171,7 @@ func (tm *TokenManager) CreateAccessToken(creds interface{}) (_ *jwt.Token, err
 		claims.Email = t.Email
 		claims.Name = t.Name
 		claims.Picture = t.Picture
+		claims.Subject = t.Subject
```

Comment:
In lines 160-174, might we annotate with comments to remind ourselves which parts of the switch get engaged where?